src/tools/genpolicy/src/mount_and_storage.rs

```diff
@@ -238,13 +238,32 @@ fn get_persistent_volume_claim_mount(
         );
     }
 
-    let is_blk_mount = pvc_resource
-        .and_then(|pvc_resource| pvc_resource.spec.storageClassName.as_ref())
-        .is_some_and(|sc| settings.common.virtio_blk_storage_classes.contains(sc));
+    let storage_class =
```

Comment:
I would prefer more a straightforward syntax instead of using and_then and map_or. Perhaps the same functionality can be achieved using some if/else statements that mere mortals like me can understand easier?

But I don't mind if you want to keep and_then, map_or, etc.